### PR TITLE
Updated GSOC Page

### PR DESCRIPTION
**Pull request by fiterace**
solves #83 please review and merge @abhigyank

### CODE REVIEW

File: gsoc/index.html [hunk 1]
```diff
@@ -70,6 +70,96 @@ <h1 class="text-dec">IIT MANDI GSOC SELECTIONS</h1>
...
+        <h1 class="centered">5+1+2 Selections in 2019</h1>
+        <hr>
+        <br>
+        <br>
+        <div class="col-lg-3 centered">
+          <img class="img img-circle" src="img/people/vipulsharma.jpg" height="120px" width="120px" alt="Vipul Sharma">
+          <br>
+          <h4><b>Vipul Sharma</b></h4>
+          <!--<a href="#" class="icon icon-twitter"></a>-->
+          <a href="https://www.facebook.com/vsvipul555?ref=br_rs" class="fab fa-facebook-f social-button"></a>
```
Comment (Varunvaruns9) [inline gsoc/index.html]: Remove `?ref=br_rs`.

File: gsoc/index.html [hunk 2]
```diff
@@ -70,6 +70,96 @@ <h1 class="text-dec">IIT MANDI GSOC SELECTIONS</h1>
...
+          <a href="https://github.com/vsvipul" class="social-button fab fa-github"></a>
+          <p>ZULIP</p>
+        </div><!-- col-lg-3 -->
+
+        <div class="col-lg-3 centered">
+          <img class="img img-circle" src="img/people/dheerajyadav.jpeg" height="120px" width="120px" alt="Dheeraj Yadav">
+          <br>
+          <h4><b>Dheeraj Yadav</b></h4>
+          <!--<a href="#" class="icon icon-twitter"></a>-->
+          <a href="https://www.facebook.com/dheeraj.168?ref=br_rs" class="fab fa-facebook-f social-button"></a>
```
Comment (Varunvaruns9) [inline gsoc/index.html]: From here also.

File: gsoc/index.html [hunk 3]
```diff
@@ -70,6 +70,96 @@ <h1 class="text-dec">IIT MANDI GSOC SELECTIONS</h1>
...
+          <a href="https://github.com/Sharadd15" class="social-button fab fa-github"></a>
+          <p>LINUX FOUNDATION</p>
+        </div><!-- col-lg-3 -->
+
+        <div class="col-lg-3 centered">
+          <img class="img img-circle" src="img/people/prajjwaljha.jpg" height="120px" width="120px" alt="Prajjwal Jha">
+          <br>
+          <h4><b>Prajjwal Jha</b></h4>
+          <!--<a href="#" class="icon icon-twitter"></a>-->
+          <a href="https://www.facebook.com/prajjwal.jha?ref=br_rs" class="fab fa-facebook-f social-button"></a>
```
Comment (Varunvaruns9) [inline gsoc/index.html]: And here.

File: gsoc/index.html [hunk 4]
```diff
@@ -70,6 +70,96 @@ <h1 class="text-dec">IIT MANDI GSOC SELECTIONS</h1>
...
+        </div><!-- col-lg-3 -->
+
+        <div class="col-lg-3 centered">
+          <img class="img img-circle" src="img/people/kartikkathuria.jpeg" height="120px" width="120px" alt="Kartik Kathuria">
+          <br>
+          <h4><b>Kartik Kathuria</b></h4>
+          <!--<a href="#" class="icon icon-twitter"></a>-->
+          <a href="https://www.facebook.com/kartik.kathuria.146" class="fab fa-facebook-f social-button"></a>
+          <a href="https://github.com/kartik1000" class="social-button fab fa-github"></a>
+          <p>(Contract) PHPMYADMIN</p>
```
Comment (Varunvaruns9) [inline gsoc/index.html]: Write Contract in all caps like MENTOR.

File: gsoc/index.html [hunk 5]
```diff
@@ -70,6 +70,96 @@ <h1 class="text-dec">IIT MANDI GSOC SELECTIONS</h1>
...
+        </div><!-- col-lg-3 -->
+
+        <div class="col-lg-3 centered">
+          <img class="img img-circle" src="img/people/gauravkumar.jpg" height="120px" width="120px" alt="Gaurav Kumar">
+          <br>
+          <h4><b>Gaurav Kumar</b></h4>
+          <!--<a href="#" class="icon icon-twitter"></a>-->
+          <a href="https://www.facebook.com/gauravkumarbtp" class="fab fa-facebook-f social-button"></a>
+          <a href="https://github.com/gaurav047" class="social-button fab fa-github"></a>
+          <p>LABLUAPUCRIO</p>
```
Comment (Varunvaruns9) [inline gsoc/index.html]: PUCRIO is the name of the university. Just keep LABLUA.

**Review by Varunvaruns9 — CHANGES_REQUESTED** [review]
I've suggested some minor changes otherwise it looks good to me.

**Comment by fiterace** [comment]
@Varunvaruns9 I have made the suggested changes. please review and merge.

**Comment by Varunvaruns9** [comment]
@abhigyank.

**Pull request merged**